cmd/pinniped/cmd/login_oidc.go

```diff
@@ -39,6 +39,18 @@ const (
 	// which specifies "cli_password" when using an IDE plugin where there is no interactive CLI available. This allows
 	// the user to use one kubeconfig file for both flows.
 	upstreamIdentityProviderFlowEnvVarName = "PINNIPED_UPSTREAM_IDENTITY_PROVIDER_FLOW"
+
+	// When using a browser-based login flow, the user may skip printing the login URL to the screen in the case
+	// where the browser was launched with the login URL. This can be useful, for example, when using a console-based
+	// UI like k9s, to avoid having any output to stderr which may confuse the UI. Set this env var to "true" to
+	// skip printing the URL.
+	skipPrintLoginURLEnvVarName = "PINNIPED_SKIP_PRINT_LOGIN_URL"
+
+	// Set this env var to "true" to cause debug logs to be printed to stderr.
+	debugEnvVarName = "PINNIPED_DEBUG"
+
+	// The value to use for true/false env vars to enable the behavior caused by the env var.
+	envVarTruthyValue = "true"
```

Comment:
Should we relax this a bit to allow for `True` or `TRUE`? The following `func` should only return true for a very limited number of values as per https://gosamples.dev/string-to-bool/. We could still document using `true` but this allows for variations.

```golang
func isTruthyString(s string) bool {
	b, err := strconv.ParseBool(s)
	return err == nil && b
}
```

Reply:
We can do that if you'd like. I was copying the style used by the existing `PINNIPED_DEBUG` env var which is either `true` or anything else is considered not true.